src/cpr_data_access/search_adaptors.py

```diff
@@ -80,14 +82,21 @@ def search(self, parameters: SearchParameters) -> SearchResponse:
         :return SearchResponse: a list of families, with response metadata
         """
         total_time_start = time.time()
+        sensitive = is_sensitive_query(parameters.query_string, SENSITIVE_QUERY_TERMS)
 
         vespa_request_body: dict[str, Any] = {
-            "yql": build_yql(parameters),
+            "yql": build_yql(parameters, sensitive),
             "timeout": "20",
             "ranking.softtimeout.factor": "0.7",
         }
+
         if parameters.exact_match:
             vespa_request_body["ranking.profile"] = "exact"
+        elif sensitive:
+            vespa_request_body["ranking.profile"] = "hybrid_no_closeness"
```

Comment:
What does hybrid no closeness mean for a ranking profile, can't see anything in the docs?

Reply:
Heres where the definition is added: https://github.com/climatepolicyradar/navigator-infra/pull/316